jury/metrics/accuracy/accuracy_for_language_generation.py

```diff
@@ -104,7 +108,11 @@ def _compute_single_pred_single_ref(
             for token, pred_count in pred_counts.items():
                 if token in ref_counts:
                     score += min(pred_count, ref_counts[token])  # Intersection count
-            scores.append(score / max(len(pred), len(ref)))
+            try:
+                scores.append(score / max(len(pred), len(ref)))
+            except ZeroDivisionError:
+                logger.warning("Empty pred/ref. Ignoring!")
```

Comment:
We can be more precise on this warning, e.g.

```python
logger.warning("Empty pred & ref after preprocess (`normalize_text`) step. Ignoring!")
```

Reply:
I also wanted to show original pred/ref texts. But that requires tracking the original lists as well. For very large samples, IMO it might hog allocate a lot of memory to maintain 2 similar lists.

Reply:
I think in parallel, no need to add these in the warning message.